editor/editor_inspector.cpp

```diff
@@ -3323,6 +3338,14 @@ void EditorInspector::_property_deleted(const String &p_path) {
 		return;
 	}
 
+	if (p_path.begins_with("metadata/")) {
+		String name = p_path.replace_first("metadata/", "");
+		undo_redo->create_action("Remove metadata");
+		undo_redo->add_do_method(object, "set_meta", name, Variant());
```

Comment:
Why not `remove_meta()`?

Reply:
Will change, but at this point there is not much reason to use set_meta() like this and we might allow Variant() as meta, but this requires changing usage everywhere we can find and including the docs. Somebody has to do this, though.